dev-client/src/screens/HomeScreen/components/MapSearch.tsx

```diff
@@ -192,7 +192,10 @@ export default function MapSearch({zoomTo, zoomToUser, toggleMapLayer}: Props) {
                 }}
                 value={query}
                 placeholder={t('search.placeholder')}
-                style={searchBarStyles.search}
+                style={{
+                  ...searchBarStyles.search,
+                  ...searchBarStyles.wideSearch,
```

Comment:
Suggestion: It feels weird to me to use both styles, especially when they both have a width property. Even though I know the latter one will override it, I'd prefer either 
- using two styles but pulling out a `standardWidthSearch` or something so the width part is separate, or 
- maybe creating a separate `wideSearch` that duplicates the other style properties, or 
- if you prefer this approach, maybe just name it `wideSearchOverride` instead of `wideSearch`, so it's clear it's not the same type of thing as `search`.